src/handlers/ens.ts

```diff
@@ -72,6 +139,42 @@ export const fetchMetadata = async ({
   }
 };
 
+export const fetchEnsTokens = async ({
+  address,
+  contractAddress = ENS_NFT_CONTRACT_ADDRESS,
+  timeAgo,
+}: {
+  address: string;
+  contractAddress?: string;
+  timeAgo: Duration;
+}) => {
+  try {
+    const { data } = await ensClient.query<EnsAccountRegistratonsData>({
+      query: ENS_ACCOUNT_REGISTRATIONS,
+      variables: {
+        address: address.toLowerCase(),
+        registrationDate_gt: Math.floor(
+          sub(new Date(), timeAgo).getTime() / 1000
+        ).toString(),
+      },
+    });
+    return data.account.registrations.map(registration => {
+      const tokenId = BigNumber.from(registration.domain.labelhash).toString();
+      const token = buildEnsToken({
+        contractAddress,
+        imageUrl: `https://metadata.ens.domains/mainnet/${contractAddress}/${tokenId}/image`,
+        name: registration.domain.name,
+        tokenId,
+      });
+      return token;
+    });
+  } catch (error) {
+    logger.sentry('ENS: Error getting ENS unique tokens', error);
+    captureException(new Error('ENS: Error getting ENS unique tokens'));
+    throw error;
```

Comment:
do we want to throw an error? maybe returning a [] to not break the flow from where it's being used

Reply:
Yep, we can make it return an empty array!